lambdaclass/mirra_backend
# [GH-669] Add new combo skill behavior

## Motivation

Some CoM characters could use combos, like Kenzu.
Skills config wasn't being used from JSON file.
Closes #669 

## Summary of changes

- [Delete skills config from json](https://github.com/lambdaclass/mirra_backend/commit/98432fcf3d1c6c7d76ea14ea88514233a0bf6004)
- [Add new skill combo fields and validation](https://github.com/lambdaclass/mirra_backend/commit/4d639f1941ebe5bf49ef286c94441caf1194fc38)
- [Add Kenzu character and combo skills in seeds](https://github.com/lambdaclass/mirra_backend/commit/1729d002ff6df7b82fe495993486b634c86606ce)
- [Add Kenzu to game client character select](https://github.com/lambdaclass/mirra_backend/commit/e45228ca54017fec8c943f0d381a8aee570c871c)
- [Add combo skills parsing and encoding](https://github.com/lambdaclass/mirra_backend/commit/792bbc24abb62f6a7a4eb50fc8ac644d00b27198)
- [Handle combo skills sequence behavior](https://github.com/lambdaclass/mirra_backend/commit/9784c4f2ec9887f2500117bafb584b7c2767ab70)
- [Update Kenzu combo skills seed params](https://github.com/lambdaclass/mirra_backend/pull/812/commits/c8b16a2412f8c56659d5377f62f72080225f6418)

⚠️ The correct Kenzu basic skill mechanics are included in https://github.com/lambdaclass/mirra_backend/pull/821

## How to test it?

Open game client and play as kenzu.
Press I and test the timing to see if you perform a combo or not.
Also test in Unity to see if nothing breaks.

## Checklist
